dialog.mdl

```diff
@@ -5372,7 +5372,9 @@ dialogs =
         (TeamPaneImage, SideRight, Point(10, -150), Point(-750, -150), .5, ofTeam | ofExpandedTeam, false),
 
         // MergedInvestmentsPaneImage: the investments pane (F5)
-        (MergedInvestmentsPaneImage, SideCenter, Point(-192, 2000), Point(-192, -265), .5, ofInvestment, false),
+        //Change from Bunnywabbit to make investor pane placement make more sense
+        //(MergedInvestmentsPaneImage, SideCenter, Point(-192, 2000), Point(-192, -265), .5, ofInvestment, false),
+        (MergedInvestmentsPaneImage, SideRight, Point(10, -265), Point(-490, -265), .2, ofInvestment, false),

```

Comment:
I think we should keep the animations consistent. I find it subconsciously confusing if every window has a different animation (length, positioning, etc.). Therefore, I suggest we keep this animation at `.5`s.